components/infiniteLogoCarousel.tsx

```diff
@@ -3,7 +3,7 @@ import { InfiniteMovingCards } from "~/components/ui/infinite-moving-cards";
 export const InfiniteLogoCarousel = () => {
   return (
     <div className=" relative flex flex-col items-center justify-center overflow-hidden rounded-md antialiased">
-      <h2 className="w-10/12 py-4 text-center text-2xl font-bold md:text-4xl">
+      <h2 className="w-10/12 py-4 md:text-center text-2xl font-bold md:text-4xl">
```

Comment:
It's best practice to also have the mobile styles in line, so add `text-left` to this className.